README.md

```diff
@@ -1,111 +1,160 @@
 # cellxgene
 
-### An interactive, performant explorer for single cell transcriptomics data.
+> an interactive performant explorer for single cell transcriptomics data
 
-<img align="right" width="350" height="218" src="./example-dataset/cellxgene-demo.gif" pad="50px">
-cellxgene is an open-source experiment in how to bring powerful tools from modern web development to visualize and explore large single-cell transcriptomics datasets.
-Started in the context of the Human Cell Atlas Consortium, cellxgene hopes to both enable scientists to explore their data and to equip developers with scalable, reusable patterns and frameworks for visualizing large scientific datasets.
+`cellxgene` is an interactive data explorer that leverages modern web development techniques to visualize large single-cell transcriptomics datasets, such as the data coming from the [Human Cell Atlas](https://humancellatlas.org). We hope to enable biologists and computational researchers to explore their data, and to demonstrate general, scalable, and reusable patterns for scientific data visualization.
 
-## Features
+<img width="500" src="./docs/cellxgene-demo.gif" pad="50px">
 
-- **Visualization at scale:** built with [WebGL](https://www.khronos.org/webgl/), [React](https://reactjs.org/) & [Redux](https://redux.js.org/) to handle visualization of at least 1 million cells.
+## features
 
-- **Interactive exploration:** select, cross-filter, and compare subsets of your data with performant indexing and data handling.
+- Built for **visualization at scale** using [WebGL](https://www.khronos.org/webgl/), [React](https://reactjs.org/), and [Redux](https://redux.js.org/) to handle at least 1 million cells.
+- Enables **interactive exploration** with performant indexing and data handling for selection, cross-filtering, and comparisons across subsets of data.
+- Support for **computational backends**  through a client-server model that can support a range of existing analysis packages for backend computational tasks (currently built for [scanpy](https://github.com/theislab/scanpy)).
 
-- **Flexible API:** the cellxgene client-server model is designed to support a range of existing analysis packages for backend computational tasks (eg scanpy), integrated with client-side visualization via a [REST API](https://restfulapi.net/).
+## getting started
 
-## Getting Started
+You'll need **python 3.6** and **Google Chrome**. Currently tested on OSX or Windows (via WSL using Ubuntu). It should work on other platforms, if you run into trouble let us know (see [help](#help-and-contact) below).
```

Comment:
I'm not sure this is clear enough about Windows testing - the front-end is tested on Windows.  The backend is Unbuntu (the fact that it is Ubuntu on WSL/Windows doesn't really mean much - more or less just a different kernel).

I would clarify by saying:
* Web UI tested on OSX and Windows, using Chrome
* Python CLI tested on Ubuntu and OSX

Reply:
fixed